sample_xblocks/filethumbs/filethumbs.py

```diff
@@ -18,14 +19,18 @@
 
 log = logging.getLogger(__name__)
 
-arrow = ["11011",
-         "10001",
-         "00000",
-         "11011",
-         "11011",
-         "11011",
-         "10001"]
-arrow = [map(int, x) for x in arrow]
```

Comment:
Fascinating. This obscure piece of code seems to be just converting strings to ints. Why not define the list with ints to begin with?